openedx_webhooks/lib/edx_repo_tools_data/test/models/test_people.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from openedx_webhooks.lib.exceptions import NotFoundError
+
+
+class TestGet:
```

Comment:
OK, it just seems odd to mix the cultures like this. There are places where you have test classes and test functions intermixed, where I can't see the logic.  With unittest, test classes are necessary, and are the basis for setUp methods, test case attributes, and so on.  These classes are giving us none of that.